api/src/main/java/org/eclipse/microprofile/lra/annotation/Compensate.java

```diff
@@ -38,6 +37,13 @@
  * </p>
  *
  * <p>
+ * In the case the ability to compensate the Long Running Action is time bounded, this is, the action can only be
+ * undone within a certain time limit when the action has started, you can limit the lifespan of the
+ * Long Running action with the {@see LRA.timeLimit} and {@see LRA.timeUnit}. With these parameters you can define
+ * The maximum lifetime of the Long Running Actions and makes it eligible for automatic cancellation.
```

Comment:
The grammar could be improved (for example there is a new sentence without a full stop). Better would be something like:
> In the case where the ability to compensate the Long Running Action is time bounded you can limit the lifespan of the Long Running action by providing values for the {@see LRA.timeLimit} and {@see LRA.timeUnit} attributes. When the time limit is reached the LRA becomes eligible for automatic cancellation.

I have a similar comment regarding the javadoc for the Complete annotation below.

Reply:
Updated